browser/css/jsdialogs.css

```diff
@@ -2024,6 +2024,11 @@ input[type='number']:hover::-webkit-outer-spin-button {
 	background-color: transparent !important;
 	border-color: transparent;
 }
+
+#toolbar-down  .jsdialog .ui-edit-container{
+	place-content: center;
```

Comment:
possibly that should be the default for any `.ui-edit-container`, but for now is fine